### ISSUE TITLE
Step 11 tutorial missing step

### ISSUE THREAD

**Issue by yarnball** [reactjs/react-router-tutorial#218]
Great tutorial!  Thank you :)

Step 11 seems to be missing something. I keep getting the error:

_Uncaught SyntaxError: Unexpected token < (bundle.js:1)_

I'm pretty sure the cause is that it's missing:

`publicPath: '/'`

from `webpack.config.js`

```
  output: {
    path: 'public',
    filename: 'bundle.js',
    publicPath: '/'
  },
```

**Comment by timdorr** [comment]
Yep, that looks to be a small oversight. Can you submit a PR to fix it?

**Comment by spik3s** [comment]
Small fix, now that `publicPath: '/'` is declared, the snippet of code

```
output: {
    path: 'public',
    filename: 'bundle.js',
    publicPath: '/'
  }
```

contains also  `path: 'public'` which should be declared later on in the tutorial. Otherwise, it will keep returning the _Unexpected token < (bundle.js:1)_ error.

**Comment by smilingkylan** [comment]
Thanks a ton for the heads-up guys.

**Comment by eurobob** [comment]
the `path: 'public'` issue that spik3s pointed out is still an issue. Tripped me up at least

**Comment by dloret** [comment]
Correct, the issue persists if webpack.config.js is not amended in a similar way at this first step of booting a prod server:
`  output: {
    /*path: "public",*/
    filename: "bundle.js",
    publicPath: "/"
  },`

@spik3s: thanks.

**Comment by merriam** [comment]
To add a bit more information:

1.  The error is from Express not finding ./public/bundle.js because it is looking for ./bundle.js.  It keeps looping back until it just tries to deal with index.html.  See http://stackoverflow.com/questions/38743275/express-js-unexpected-token.

2.  Moving bundle.js from ./public to . will make the error disappear.

3.  One way to move bundle.js is not to put in the "path: 'public'" line until after this step.  This also keeps the public directory from being created, which makes the next step of the tutorial break.

The only remaining issue is updating the path in webpack .config.js is now listed twice, so those following the tutorial will still get stuck.  Adding a PR.

**Comment by merriam** [comment]
If we apply PR #310, we can close #229, #242, #280, #282, #289, and #291.  Everyone runs across this bug.
